prez/dependencies.py

```diff
@@ -162,9 +173,10 @@ async def load_annotations_data_to_oxigraph(store: Store):
     g = Dataset(default_union=True)
     for file in (Path(__file__).parent / "reference_data/annotations").glob("*"):
         g.parse(file)
+    # Whats the point of loading this into an RDFLib dataset them serializing it again
+    # to then load it into the oxigraph store?
```

Comment:
No point - probably a copy paste error, can be simplified

Reply:
This is resolved in https://github.com/RDFLib/prez/pull/384/commits/455bb746fd982bb446593eb001857fcbef34e567